src/js/components/bulkDownload/accounts/UserSelections.jsx

```diff
@@ -120,10 +120,10 @@ export default class UserSelections extends React.Component {
 
     generateFyString() {
         const { fy, quarter, period } = this.props.accounts;
-        const quarterSelection = quarter === '' ? `P${period}` : `Q${quarter}`;
-        if (fy && quarterSelection) {
+        const timePeriodSelection = quarter ? `Q${quarter}` : `P${period}`;
+        if (fy && timePeriodSelection) {
```

Comment:
Do we need `timePeriodSelection` in  this if statement `(fy && timePeriodSelection)`?

Since `timePeriodSelection` will always evaluate to true since is will always be at the very least a string with either a `Q` or `P`.

Reply:
No